pkg/vtpm/src/server.cpp

```diff
@@ -101,13 +263,20 @@ int readMessage(int sock, google::protobuf::uint32 size)
             output_file.close();
         }
     }
-    size = response.ByteSize() + CODED_STRM_HDR_LEN;
-    char *resp_buffer = new char [size];
-    google::protobuf::io::ArrayOutputStream aos(resp_buffer, size);
-    CodedOutputStream *coded_output = new CodedOutputStream(&aos);
-    coded_output->WriteVarint32(response.ByteSize());
-    response.SerializeToCodedStream(coded_output);
-    send(sock, (void *)resp_buffer, size, 0);
+    //Clear all the files: inputFiles and expectedFiles
```

Comment:
Presumably you need to do this cleanup of the files even if there was an error.

Reply:
fixed it in the latest diff.